src/main/java/org/sagebionetworks/bridge/reporter/helper/BridgeHelper.java

```diff
@@ -26,6 +28,10 @@
 public class BridgeHelper {
     private static final Logger LOG = LoggerFactory.getLogger(BridgeHelper.class);
 
+    // match read capacity in ddb table
+    static final long MAX_PAGE_SIZE = 30L;
```

Comment:
To clarify, "matching" the capacity we currently have in the DDB table is the wrong approach. The capacity was set that high because that's how much capacity we're currently using. Most likely, whatever capacity you set here, we will need to increase both the table and the index by that much.

While we *can* increase our capacity by that much (it would only cost $6/mo or so), the question is, do we need to? This job only runs once a day at most, and is not time critical. Setting the page size to something like 10 would still almost certainly allow us to complete the job in 5-10 min or so.